xwiki-platform-core/xwiki-platform-livetable/xwiki-platform-livetable-ui/src/main/resources/XWiki/LiveTableResultsMacros.xml

```diff
@@ -30,8 +30,8 @@
   <parent>XWiki.WebHome</parent>
   <author>xwiki:XWiki.Admin</author>
   <contentAuthor>xwiki:XWiki.Admin</contentAuthor>
-  <date>1465822369000</date>
-  <contentUpdateDate>1465822369000</contentUpdateDate>
+  <date>1465895028000</date>
```

Comment:
This is noise added to every commit, so it shouldn't be committed. Do you know about `git add --interactive`?